tools/cldr-apps/js/src/esm/cldrText.js

```diff
@@ -467,7 +467,7 @@ const strings = {
   special_users: "Users",
   special_vetting_participation: "Vetting Participation",
   special_vetting_participation2: "Vetting Participation v2",
-  special_vsummary: "Priority Items Summary (slow)",
```

Comment:
Maybe when CLDR-14543 lands we can change this to `(fast)` :)

Reply:
good idea